Include/pyport.h

```diff
@@ -182,11 +182,10 @@ typedef Py_ssize_t Py_ssize_clean_t;
  */
 
 #if defined(_MSC_VER)
-#  if defined(PY_LOCAL_AGGRESSIVE)
+#  if defined(PY_LOCAL_AGGRESSIVE) && !defined(Py_DEBUG)
    /* enable more aggressive optimization for MSVC */
-   /* active in both release and debug builds - see bpo-43271 */
-#  pragma optimize("gt", on)
-#endif
+   /* MSBuild scripting is preferable to optimize pragmas in headers */
```

Comment:
This block now has nothing in it?

Reply:
We can remove the block and `PY_LOCAL_AGGRESSIVE`. I left them just in case (`_forceinline`?).

Reply:
I'll remove them.

Reply:
Removing the pragma here also improves the build performance of `_freeze_module` on non-debug mode.